tests/tests_fabric/strategies/test_fsdp_integration.py

```diff
@@ -164,3 +166,25 @@ def test_setup_with_orig_params_and_multiple_param_groups():
         # A regular parameter as a view into the flattened parameters
         assert isinstance(layer.weight, torch.nn.Parameter)
         assert not isinstance(layer.weight, FlatParameter)
+
+
+@RunIf(min_cuda_gpus=2, skip_windows=True, standalone=True, min_torch="2.0.0")
+@mock.patch.dict(os.environ, {})
+@pytest.mark.parametrize("compile_after_setup", [False, True])
+def test_compile(compile_after_setup):
```

Comment:
This also needs some skips as the tests in https://github.com/Lightning-AI/lightning/blob/master/tests/tests_pytorch/utilities/test_compile.py#L39-L41

Reply:
done

Reply:
Missed macos darwin skip

Reply:
It is redundant, since FSDP won't work on macos/cpu and we have min_cuda_gpus=2